containers/azure-functions-pwsh-7/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM mcr.microsoft.com/azure-functions/dotnet:3.0-dotnet3-core-tools
```

Comment:
There seems to be a `mcr.microsoft.com/azure-functions/powershell:3.0-powershell7-core-tools` container. Have you tried that one?

Reply:
Of course there is.

the dotnet one I was using is using 7.0.2 whereas the powershell 7 one is using 7.0.1 but 7.0.3 is tagged as the lts so would that not be a better dependency going forward until 7.1.0?

Updated the dockerfile to use that base image instead.